### DISCUSSION TITLE
Support for chapters

### DISCUSSION THREAD

**Discussion by codeanator** [CodyBerenson/PGMA-Modernized#88] [Show and tell]
👋 

Besides fixing IAFD scraping, I am working on another feature : chapter support.
You can find the WIP here: https://github.com/codeanator/PGMA-Modernized/tree/chapters

Features : 
- Creating chapters based on scenes
- Support for AEBN, GayHotMovies and GayDVDEmpire (GayDVDEmpire currently broken)
- Support for titles split in multiple files (see: https://support.plex.tv/articles/naming-and-organizing-your-movie-media-files/#toc-2)

More details : 
- Chapters are created only if the delta between the length of the file and the length of the title reported on the website is less than 100s
- If the total length of scenes is less than the length of the title, we assume the delta is due to intro/opening at the beginning of the file
- The scenes are names "Stars (Acts)" with fallback "Scene/Chapter Number (Acts)" for AEBN and GayHotMovies. Only scene number for GayDVDEmpire

**Comment by JPH71** [comment]
perfect....

that sounds good - liaise with Cody too!

Jason xxx
